setup.cfg

```diff
@@ -21,6 +23,10 @@ ignore =
        E125
        # continuation line over-indented for visual indent
        E127
+       # line break after binary operator (contradicts W503) 
+       W504
+       # module level import not at top of file (poblem with ROOT)
+       E402
```

Comment:
E402 what if you just flag the lines of ROOT as false positives

Reply:
Done!